sdk-utils/src/main/kotlin/com/rakuten/tech/mobile/sdkutils/eventlogger/Event.kt

```diff
@@ -0,0 +1,31 @@
+package com.rakuten.tech.mobile.sdkutils.eventlogger
+
```

Comment:
We can add the param names doc link inside class definition to understand the field easily, something like:
/**
 * Documentation: https://..........
 */

Reply:
Thanks @mdusmangani-jp, as this repository is public we can't refer to internal links.